feature/interface/singleton/otg_tests/singleton_test/README.md

```diff
@@ -30,6 +31,14 @@ a new testbed configuration with the desired port types.
 
 [^1]: The MTU specified above refers to the L3 MTU, which is the payload portion
     of an Ethernet frame.
+### TC2 - link flaps:
+* Bring down the physical layer of ATE port-1, and bring it back up.
+    Repeat this a few times (minimum 2)
```

Comment:
if there any verification needed inbetween up/down transition?  Just wait for ack from ATE and repeat?  Is the ATE call blocking or asynchronous?  Just want to make sure the ATE actually brings down the physical layer.  

2 times seems like a small number?  Is there an operational issue/target we can reference?

Reply:
I will add a verification step on the DUT/ATE for every state transition. this would be a blocking call. I have left the transition value to a min. value of 2, and we could increase this, if we identify a target reference later.